src/frontends/tensorflow/src/decoder_proto.cpp

```diff
@@ -279,12 +279,15 @@ size_t DecoderProto::get_input_size() const {
 void DecoderProto::get_input_node(size_t input_port_idx,
                                   std::string& producer_name,
                                   size_t& producer_output_port_index) const {
-    // TODO: handle body graph nodes with a couple of columns
+    // Body graph nodes may have two colons `:`, for example,
```

Comment:
Can it have more then 2 colons?

Reply:
I never met more 2 colons, but the main thing is to cut off the middle part because I have no idea how it is used.